### ISSUE TITLE
missing redis as production dependency

### ISSUE THREAD

**Issue by JSteunou** [mjackson/then-redis#31]
all said ;)

**Comment by mjackson** [comment]
@JSteunou [It's listed in `peerDependencies`](https://github.com/mjackson/then-redis/blob/4bff943af45b1bbaad245ad05f7ce90918ad054c/package.json#L24), which should install it in your production environment.

**Comment by JSteunou** [comment]
Ho miss that. Let me try again.

**Comment by JSteunou** [comment]
My bad, it works very well!
